### PR TITLE
Create 2023-02-22.md

### PR DESCRIPTION
**Pull request by jmertic**

### CODE REVIEW

File: meetings/2023-02-23.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 parent: Meetings
-title: "2023-03-23"
+title: "2023-02-23"
```
Comment (jfpanisset) [inline meetings/2023-02-23.md:3]: Meeting date was 2023-02-22, filename should be changed as well to reflect?

File: meetings/2023-02-23.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
-# AWSF TAC Meeting - Month DD, Year
+# AWSF TAC Meeting - February 23, 2023
```
Comment (jfpanisset) [inline meetings/2023-02-23.md:6]: Meeting date was 2023-02-22

File: meetings/2023-02-23.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- [  ] Larry Gritz - Sony Pictures Imageworks
+- [  ] Mark Visser - Unity Technologies
+- [  ] Matthew Low - DreamWorks Animation
+- [  ] Michael B. Johnson - Apple Inc.
+- [  ] Sean Looper - Amazon Web Services, Inc.
+- [  ] Sean McDuffee - Intel Corporation
+- [  ] Sean O'Connell - Advanced Micro Devices (AMD)
+- [  ] Tony Micilotta - DNEG
+
+### Project Representatives
```
Comment (jfpanisset) [inline meetings/2023-02-23.md:28]: Should this section list the project affiliation rather than the company, or perhaps both?

**Review by jfpanisset — DISMISSED** [review]
Want to merge this as is and I can fix up dates / who was present based on Zoom recording?

**Review by jfpanisset — DISMISSED** [review]
Looks good, let's merge it

**Comment by jmertic** [comment]
@jfpanisset I'm waiting to get the list of attendees and then will merge. Opened ticket with the LFX team

**Review by jfpanisset — APPROVED** [review]
Looks good, should we merge?

**Pull request merged**